README.md

```diff
@@ -6,14 +6,13 @@ This repository contains the list of communities and job portals you can join an
 
 ## Roadmaps:
 <p align="center"><img align="center" alt="Coding" src="https://media.giphy.com/media/xUySTOigOUHucl3rfW/giphy.gif"></p>
-<p  align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/Backend-Development#readme"><B>Backend-Development</B></a></p>
-<p  align="center"><a href=""><b>Blockchain</b></a></p>
-<p  align="center"><a href=""><b>DevOps</b></a></p>
-<p  align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/DevRel#readme"><b>DevRel</b></a></p>
-<p  align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/Frontend-Development#readme"><b>Frontend-Development</b></a></p>
-<p  align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/Fullstack-Development#readme"><b>Fullstack-Development</b></a></p>
-<p  align="center"><a href=""><b>Android-Development</b></a></p>
-<p align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/Open-Source#readme"><b>Blockchain</b></a></p>
+<p  align="center"><a href="Backend-Development#readme"><B>Backend-Development</B></a></p>
```

Comment:
Please align the roadmap names to the left. It doesn't look visually aligned with the current readme design